public/test/utils/regions/story.js

```diff
@@ -13,10 +16,28 @@ class Story {
         return group(1, this.dom, this);
     }
 
-    pacakgeSelector() {
-        return this.dom.querySelector('.front-selector');
+    getPackageInSelector(id) {
+        return $('.front-selector option[value=' + id + ']');
     }
-}
+
+    getPackageInSelectorText(id) {
+        return textInside('.front-selector option[value=\"' + id + '\"]');
+    }
+
+    selectedPackageName() {
+        return textInside('.front-selector option:selected');
+    }
+
+    manageLatestPackage(index) {
+        const selectorIndex = index + 1;
+        const selector = $('.front-selector option:eq('+ selectorIndex +')');
+        selector.prop('selected', true);
+        $(this.dom.querySelector('.front-actions')).click();
```

Comment:
Instead of mixing jquery and vanilla js it's better doing `$('.front-actions', this.dom).click()`